lib/mix/test/test_helper.exs

```diff
@@ -238,7 +238,7 @@ unless File.dir?(target) do
   """)
 
   File.cd!(target, fn ->
-    System.cmd("git", ~w[init])
+    System.cmd("git", ~w[-c core.defaultBranch=main init])
```

Comment:
Your change does not fix the issue, can you try this:

```suggestion
    System.cmd("git", ~w[-c init.defaultBranch=main init])
```

You can also set it here: https://github.com/elixir-lang/elixir/pull/10855/files#diff-e360ffde0aa6722b0c98571cdfd8927c23df50e2b7a7c3314ccca42e3955a977R227.

Reply:
Ah, yeah. Will update in a second.

Reply:
Fixed